ovp.LucyBot.yml

```diff
@@ -557,3 +567,82 @@ workflowNavigation:
   - title: Contact Us
     icon: assets/img/icons/contact.png
     isContact: true
+
+iosNavigation:
+
+  - title: "Getting Started with the Player SDK for iOS"
+  markdownURL: /assets/documentation/player-portal/ios/getting-started-ios.html
+  path: "/player/ios/getting"
+  discussionURL: null
+
+  - title: "Supported Devices for iOS"
+  markdownURL: /assets/documentation/player-portal/ios/supported-devices-ios.html
+  path: "/player/ios/supported-devices-ios.html"
+  discussionURL: null
+
+- title: "Player Events for iOS"
```

Comment:
the dash on this needs to be aligned with the above, ditto below

Reply:
fixed